### PR TITLE
Support per widget setting for two sided and set it to true when duplicating to multimirror

### PR DESCRIPTION
**Pull request by andybak**
There's no UI to change this manually at the moment but it's "doing the right thing" in terms of behaving intuitively and not breaking existing behaviour.

### CODE REVIEW

File: Assets/Scripts/Commands/MultimirrorDuplicateCommand.cs [hunk 1]
```diff
@@ -81,13 +81,22 @@ public MultimirrorDuplicateCommand(TrTransform xf, bool stampMode, BaseCommand p
...
             foreach (var widget in m_SelectedWidgets)
             {
+                if (widget is Media2dWidget)
+                {
+                    ((Media2dWidget)widget).TwoSided = true;
+                }
                 TrTransform widgetTransform_GS = TrTransform.FromTransform(widget.transform);
                 TrTransform tr_GS;
                 var xfCenter_GS = TrTransform.FromTransform(PointerManager.m_Instance.SymmetryWidget);
+                // Generally speaking we want both sides of 2d media to appear
```
Comment (mikeage) [inline Assets/Scripts/Commands/MultimirrorDuplicateCommand.cs:91]: Maybe I'm missing something obvious, but what's the difference between this boolean and the `.TwoSided` set a few lines earlier?

File: Assets/Materials/ReferenceImage.mat [hunk 2]
```diff
@@ -2,14 +2,18 @@
...
-  serializedVersion: 6
+  serializedVersion: 8
   m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
   m_PrefabInstance: {fileID: 0}
   m_PrefabAsset: {fileID: 0}
   m_Name: ReferenceImage
   m_Shader: {fileID: 4800000, guid: 8c9bb7751e2a42747aa2a55e4108bde8, type: 3}
-  m_ShaderKeywords: _ALPHATEST_ON _LIGHTMAPPING_DYNAMIC_LIGHTMAPS _UVSEC_UV1
+  m_ValidKeywords: []
```
Comment (mikeage) [inline Assets/Materials/ReferenceImage.mat:12]: was this change intended?

**Review by mikeage — COMMENTED** [review]
I'm not quite sure I followed the logic here, but I might be missing something very obvious

File: Assets/Materials/ReferenceImage.mat (hunk 2 above)
Comment (andybak) [reply Assets/Materials/ReferenceImage.mat:12]: No. I tend to assume Unity is doing the right thing when it modifies materials in response to changes I made in the UI - but this oe does look a little sus.

Comment (andybak) [reply Assets/Materials/ReferenceImage.mat:12]: I think we're going to have to trust Unity on this one - that setting doesn't correlate with anything in the UI - or with anything I changed in the shader - so it might just be an old property that Unity is pruning.

Comment (mikeage) [reply Assets/Materials/ReferenceImage.mat:12]: ok

File: Assets/Scripts/Commands/MultimirrorDuplicateCommand.cs (hunk 1 above)
Comment (andybak) [reply Assets/Scripts/Commands/MultimirrorDuplicateCommand.cs:91]: Good catch. That's left over from a small refactor.

**Review by mikeskydev — APPROVED** [review]
